test/integration/default/default_spec.rb

```diff
@@ -11,7 +11,7 @@
 
 # TODO: change once gem resource handles alternate path to `gem` command
 describe command('/opt/chef/embedded/bin/gem list --local -a -q inspec | grep \'^inspec\' | awk -F"[()]" \'{printf $2}\'') do
```

Comment:
we should use the new method once its merged to inspec, see https://github.com/chef/inspec/pull/1596